src/commands/dev/hook.js

```diff
@@ -0,0 +1,65 @@
+const execa = require("execa");
+const Command = require("@netlify/cli-utils");
+const { track } = require("@netlify/cli-utils/src/utils/telemetry");
+const {
+  // NETLIFYDEV,
+  NETLIFYDEVLOG,
+  // NETLIFYDEVWARN,
+  NETLIFYDEVERR
+} = require("netlify-cli-logo");
+
+class HookCommand extends Command {
+  async run() {
+    const { site, api, config } = this.netlify;
+
+    if (!(config && config.dev && config.dev.hooks)) {
```

Comment:
confusing to run `netlify dev:hook foo` but then inside toml i need to say `[dev.hooks]`. how to standardize this? or do we just build in tolerance for plurals?

Reply:
Hmm, not sure that's confusing in practise?

`netlify dev:hook SomeHook` runs one specific hook

In the toml there might be multiple:

```
[dev.hooks] 
  Contentful = "npm run contentful-fetch"
  ERP = "npm run fetch-from-erp"
```

Reply:
ok